puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/pipelines.groovy

```diff
@@ -0,0 +1,25 @@
+def installPipelineVars(version, os, type) {
+    playBook = [
```

Comment:
Nit: I think this could just be `playbook` as thats how the term looks in Ansible vs `play book`

Reply:
I'm pretty sure I copied this from something else. I was actually thinking about naming it vars or something